### ISSUE TITLE
Xterm window not shows up

### ISSUE THREAD

**Issue by dfantonio** [v1s1t0r1sh3r3/airgeddon#144]
#### What is your airgeddon version?
v8.01

#### What is your Linux O.S. and which version?

Ubuntu 17.10

#### Which is the chipset of your wireless card?

Qualcomm Atheros AR9485

#### Describe the issue and the steps to reproduce it

I execute the file and airgeddon loads up smoothly, I choose my interface and puts it in monitor mode. But, whenever I go to an option that must open a xterm terminal, it opens minimized, and not show up, no matter how I try to select it. 
The strange part is if I call xterm from a terminal it appears normally, so i guess it's some conflict in airgeddon. I also tried reinstalling, but nothing changed.

**Comment by v1s1t0r1sh3r3** [comment]
Do you have xset command available. I think is in x11-xserver-utils package. You can install it. With it installed, airgeddon will check if X window system is available.

Anyway, depending of your X window system configuration, you should need to launch `xhosts +` command first.

Can you put a screenshot?

**Comment by dfantonio** [comment]
Sorry, but I don't know if I got what you asked right. Here is what I tried to do:
tried to install x11-xserver-utils and then tried to launch airgeddon with xhost.
![screenshot from 2018-06-09 16-43-08](https://user-images.githubusercontent.com/40124768/41195486-7d0c9d24-6c04-11e8-98a6-4091211865ef.png)

**Comment by v1s1t0r1sh3r3** [comment]
The command is `xhost +` don't forget the plus symbol. And after launching that command, launch airgeddon as normal.

Anyway, the screenshot I'm requesting is the window problem.

**Comment by dfantonio** [comment]
I typed `xhost +` and launched airgeddon, but nothing changed. I forgot to mention before, but i didn't manage to install beef, even though I followed the official tutorial.
I recorded a video instead of a screenshot, because i think it shows more about my problem (https://www.youtube.com/watch?v=qEG7WhGtgGY&feature=youtu.be)

**Comment by v1s1t0r1sh3r3** [comment]
BeEF is a tricky thing to install it manually. It has a lot of ruby gems dependencies. I did it sometimes and is not easy. All I can say about that is recommend to you to use a Linux distribution which comes with BeEF directly installed like Parrot Security or Kali Linux.

Regarding your problem on X window system. I saw at your video that the validation is passing. So the X window system is available and the resolution is well detected. So airgeddon validations are good and passed. Not sure about what is happening and why your xterm windows is opening in a strange way minimized/hidden. I never saw that and I'm not able to reproduce it on my Ubuntu Linux so I think I can't help here. Anyway it not seems related to airgeddon. It sounds more like a O.S. problem. Try on a clean installed Ubuntu or on a Virtual machine. I'll let this issue open for 7 days, maybe somebody can help you. Good luck!

**Comment by Luan7805** [comment]
Just a hunch, but ...
It looks like you have more than one screen (or your system thinks it has) and airgeddon is displaying the xterm window on that other screen. It would be interesting to check your screen settings.

Anyway you can still close the xterm window by ubuntu-dock so airgeddon can proceed normally

**Comment by dfantonio** [comment]
First of all sorry for the delay in answering. You actually saved me Luan, you mentioned about my resolution and when I went to the settings there was a second monitor (Unknown Display) and guess what, they were configured to join. As soon as I changed that to single the xterm started working normally.
![screenshot from 2018-06-12 01-01-32](https://user-images.githubusercontent.com/40124768/41269568-32a4c5de-6ddc-11e8-95f2-cfacf49f5a92.png)

Big thanks for both of you by helping me solving this :D
